monicaleep/member-fe-microsite
# [chore]: Add Cypress

- Add and configure [cypress](https://docs.cypress.io/guides/overview/why-cypress)
- Add scripts to run tests and webserver in non-watch mode
- Update readme to document how to write tests

Verification Steps
- pull down this branch, run `npm install`
- follow the testing section in readme to a T
- you should be able to run tests both in head and headless mode
- if you are not able to, provide feedback in this PR
